cilium/cmd/config.go

```diff
@@ -77,12 +78,26 @@ func configDaemon(cmd *cobra.Command, opts []string) {
 		dumpConfig(resp.Configuration.Mutable)
 		fmt.Printf("%-24s %s\n", "k8s-configuration", resp.K8sConfiguration)
 		fmt.Printf("%-24s %s\n", "k8s-endpoint", resp.K8sEndpoint)
+		fmt.Printf("%-24s %s\n", "PolicyEnforcement", resp.PolicyEnforcement)
```

Comment:
We already have a `Policy` flag. What will be the meaning of this one? I think we should rename it to `PolicyEnforcementMode` or `PolicyMode`
```
$ cilium config
Conntrack                Enabled
ConntrackAccounting      Enabled
ConntrackLocal           Disabled
Debug                    Enabled
DropNotification         Enabled
Policy                   Disabled
PolicyTracing            Disabled
k8s-configuration        /var/lib/cilium/cilium.kubeconfig
k8s-endpoint             http://
```